documentation/tutorials/check.md

```diff
@@ -99,6 +99,39 @@ We can then run `check` with the `-drift` flag:
 
 This creates another pair of HTML and JSON reports. This time the files contain a drift report, showing how the target database schema (in this case `foobar`) differs from whats contained in the applied migrations.
 
+### Programmatically identifying drift
+<h4> <i class="fa fa-windows"></i> Windows </h4>
+
+Powershell script to parse the output JSON,find the last drift report it contains and report the status of the flag
+
+<pre class="console">
```

Comment:
To make the script clearer, instead of using `<pre class="console">...</pre>`, we could use a markdown script block:
```
    ```powershell
     ...
    ```
```
That gives syntax highlighting:

![image](https://user-images.githubusercontent.com/60435811/196377619-ef5ae8cd-8396-455f-b619-23a1fe17b664.png)

Could also do the same for the bash script

Reply:
The same would apply to the bash section

Reply:
good idea